### PR TITLE
fix. logical error. Reading command line arguments.

### PR DESCRIPTION
**Pull request by le9i0nx**
#18

### CODE REVIEW

File: moira/config.py [hunk 1]
```diff
@@ -95,3 +93,7 @@ def read():
             CHECK_INTERVAL = cfg['checker'].get('check_interval', 5)
             METRICS_TTL = cfg['checker'].get('metrics_ttl', 3600)
             STOP_CHECKING_INTERVAL = cfg['checker'].get('stop_checking_interval', 30)
+
+    HTTP_PORT = args.port
```
Comment (gmlexx) [inline moira/config.py]: What would happen if I don't set the log and port from command line?

File: moira/config.py [hunk 2]
```diff
@@ -73,8 +73,6 @@ def read():
 
     ARGS = args
     CONFIG_PATH = args.c
```
Comment (gmlexx) [inline moira/config.py]: What happen now, when you have removed default value from args.c?

**Comment by gmlexx** [comment]
Did you try to run checker or api, after your changes?

File: moira/config.py (hunk 2 above)
Comment (le9i0nx) [inline moira/config.py]: ))

**Comment by le9i0nx** [comment]
no
I am a bad assistant.
))

**Comment by gmlexx** [comment]
Let's make three more steps:

- [ ] Move config assigments to one place. Now args.c and others args.* are splitted.
- [ ] Squash commits to one
- [ ] Manual test

**Comment by le9i0nx** [comment]
1. this was done deliberately.
goal in the shell should receive the highest priority
I think I can hide the code in the function.
https://github.com/le9i0nx/worker/blob/23eaec3002cf42c09fd8b42ac17a71a177a708e3/moira/config.py#L78-L96

2. I do not understand that I have to do here.
3. Set an example in the project. So that I could write the necessary code.

**Comment by gmlexx** [comment]
1. Sorry, my mistake. All ok.
2. I ment rewrite git history to make one commit instead 5.
3. Manual means just build and run manually. I can do it, but later.

**Comment by le9i0nx** [comment]
2. merged into one.
3. I checked with the help of the main function in config.py

**Comment by beevee** [comment]
Hi! Thanks for your contribution. I'd like to mention you in our AUTHORS file, but I can't find your name. If it's ok, please tell us your name.

**Comment by le9i0nx** [comment]
#30

**Pull request merged**